### ISSUE TITLE
Make edm::loadCap properly report missing dictionaries

### ISSUE THREAD

**Issue by Dr15Jones** [cms-sw/cmssw#14242]
If ROOT knows the name of a class but does not have a dictionary for it, `edm::loadCap` will throw the exception

```
An exception of category 'DictionaryNotFound' occurred while
  [0] Constructing the EventProcessor
  [1] Constructing input source of type NewEventStreamFileReader
Exception Message:
No TClass for class: 'std::type_info*'
```

The name of the missing class is not reported correctly.

The traceback for when this happened was `edm::loadCap` -> `edm::loadType` -> `edm::loadType` -> edm::checkClassDictionaries`.

The class in question was `BXVector<l1t::GlobalAlgBlk>` which had an internal missing dictionary of `std::vector<GlobalAlgBlk>`.

**Comment by Dr15Jones** [comment]
assign core

**Comment by wddgit** [comment]
This is fixed by PR #14896

**Comment by wddgit** [comment]
PR #14896 was closed and never approved. This was really fixed by PR #15198, which was merged into 8_1_X on 27 July 2016.

**Comment by Dr15Jones** [comment]
+1
